README.md

```diff
@@ -7,7 +7,7 @@
 ## Installation
 
 ```bash
-$ yarn install
+$ yarn install && yarn add @tonomy/tonomy-id-sdk@development
```

Comment:
missing `--no-lockfile` flag

Reply:
for yarn it's working without --no-lockfile otherwise it throw error